app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -124,6 +124,17 @@ export default class AMRHeatmapView extends React.Component {
     return maxValues;
   }
 
+  hasDataToDisplay(maxValues) {
+    let hasData = false;
+    Object.entries(maxValues).forEach(metric => {
+      const value = metric[1];
+      if (value > 0) {
+        hasData = true;
+      }
+    });
+    return hasData;
```

Comment:
I think you can do `return some(maxValue => maxValue > 0, maxValues)`

But this implicitly assumes that all metrics only allow for positive numbers. This is true right now, but if for some reason we allow a metric with negative values in the future, a bug can sneak in that's hard to catch at that point.

Would `return some(sample => size(sample.amrCount) > 0, samplesWithAmrCounts)` work instead?

This checks if there's at least one sample that has a non-empty amrCount object (more than zero keys in the object).

(granted, for negative numbers the Legend would probably also break)